docs/modules/rediscache.md

```diff
@@ -25,10 +24,24 @@ Provisions a new Redis cache.
 |----------------|------|-------------|----------|---------------|
 | `location` | `string` | The Azure region in which to provision applicable resources. | Required _unless_ an administrator has configured the broker itself with a default location. | The broker's default location, if configured. |
 | `resourceGroup` | `string` | The (new or existing) resource group with which to associate new resources. | N | If an administrator has configured the broker itself with a default resource group and nonde is specified, that default will be applied, otherwise, a new resource group will be created with a UUID as its name. |
+| `skuCapacity` | `integer` | The size of the Redis cache to deploy.  Valid values: for C (Basic/Standard) family (0, 1, 2, 3, 4, 5, 6), for P (Premium) family (1, 2, 3, 4). | N | If not provided, `0` is used for C (Basic/Standard) family; `1` is used for P (Premium) family. |
```

Comment:
Is there a unit of measurement that we can mention in this description? GB?

Reply:
Detailed cache size can be found [here](https://azure.microsoft.com/en-us/pricing/details/cache/). I'll add it later.